test/e2e/e2e_test.go

```diff
@@ -18,11 +18,16 @@ package e2e
 
 import (
 	"testing"
+	"flag"
 
 	"github.com/kubernetes-incubator/service-catalog/test/e2e/framework"
 )
 
+var brokerImage string
```

Comment:
Yea, I know why that happens. You have to push the old commit, open and then push the new commit. You can't push to a closed branch and then reopen it.